app/src/main/java/com/example/playlistmaker/player/ui/activity/FragmentPlayer.kt

```diff
@@ -2,33 +2,53 @@ package com.example.playlistmaker.player.ui.activity
 
 import android.os.Bundle
 import android.util.Log
+import android.view.LayoutInflater
 import android.view.View
-import androidx.appcompat.app.AppCompatActivity
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
 import androidx.lifecycle.lifecycleScope
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 import com.bumptech.glide.Glide
 import com.bumptech.glide.load.resource.bitmap.RoundedCorners
 import com.example.playlistmaker.R
-import com.example.playlistmaker.databinding.ActivityPlayerBinding
+import com.example.playlistmaker.databinding.CreatingAlbumAlertBinding
+import com.example.playlistmaker.databinding.FragmentPlayerBinding
+import com.example.playlistmaker.media.data.Playlist
+import com.example.playlistmaker.media.data.PlaylistsBottomAdapter
+import com.example.playlistmaker.media.data.PlaylistsState
 import com.example.playlistmaker.player.domain.Track
 import com.example.playlistmaker.player.ui.PlayerState
-import com.example.playlistmaker.player.ui.view_model.PlayerViewModel
-import com.example.playlistmaker.player.ui.view_model.PlayerViewModel.Companion.PLAYER_BUTTON_PRESSING_DELAY
+import com.example.playlistmaker.player.ui.view_model.FragmentPlayerViewModel
+import com.example.playlistmaker.player.ui.view_model.FragmentPlayerViewModel.Companion.PLAYER_BUTTON_PRESSING_DELAY
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.snackbar.Snackbar
 import kotlinx.coroutines.Job
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.launch
 import org.koin.androidx.viewmodel.ext.android.viewModel
 
-class PlayerActivity : AppCompatActivity() {
+class FragmentPlayer : Fragment() {
```

Comment:
🍏 В андроид разработке принято называть фрагменты в таком порядке: 
```suggestion
class PlayerFragment : Fragment() {
```